README.md

```diff
@@ -2,15 +2,52 @@
 [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/ome/omero-analysis-desktop/master?filepath=napari.ipynb)
 
 
-Run [OMERO clients](https://www.openmicroscopy.org/omero/downloads/) and [Napari](http://napari.org/) in a Linux desktop using Jupyter.
+Run [OMERO clients](https://www.openmicroscopy.org/omero/downloads/), [Napari](http://napari.org/) and [omero-napari](https://gitlab.com/openmicroscopy/incubator/omero-napari) in a Linux desktop using Jupyter.
 
 This is based on https://github.com/ryanlovett/nbnovnc
 
 Either [run on mybinder.org](https://mybinder.org/v2/gh/jburel/omero-analysis-desktop/master) or build locally with [repo2docker](https://repo2docker.readthedocs.io/):
+
+To build locally:
+
+ * Install [Docker](https://www.docker.com/) if required
+ * Create a virtual environment and install repo2docker from PyPI.
+ * Clone this repository
+ * Run  ``repo2docker``. 
+ * Depending on the permissions, you might have to run the command as an admin
+
 ```
+pip install jupyter-repo2docker
+git clone https://github.com/ome/omero-analysis-desktop.git
+cd omero-analysis-desktop
 repo2docker .
 ```
 
-Open the displayed URL, then go to `/desktop` e.g. http://localhost:8888/desktop and you should see a Linux desktop with icons for OMERO.insight and FIJI.
+Open the displayed URL, then go to `/desktop` in the menu option on the right-hand side.
+You should see a Linux desktop with icons for OMERO.insight and FIJI.
+
+Use omero-napari
+~~~~~~~~~~~~~~~~
+
+Once the desktop is open, open a terminal run:
+
+```
+conda init bash
+```
+
+Close the terminal, re-open a terminal and activate the conda environment:
+
+```
+conda activate notebook
+```
+
+In the same terminal you can then use the OMERO CLI client ``omero-napari`` for example:
+
+```
+omero napari view --server idr.openmicroscopy.org --user public --password public --zarr --endpoint_url https://s3.embassy.ebi.ac.uk/ Image:6001240
```

Comment:
This doesn't work, you might need to move the login arguments (or login separately)
![image](https://user-images.githubusercontent.com/1644105/80588001-72008280-8a0f-11ea-91f8-59f9ffac5213.png)


In addition this will need to be `wss://idr.openmicroscopy.org/omero-ws` to work on mybinder